frontend/app/routes/$lang/_protected/access-to-governmental-benefits/view.tsx

```diff
@@ -100,93 +100,70 @@ export default function AccessToGovernmentalsBenefitsView() {
-          </InlineLink>
-        </div>
-
-        <fetcher.Form method="post" noValidate>
-          <input type="hidden" name="_csrf" value={csrfToken} />
-          <div className="flex flex-wrap items-center gap-3">
-            <ButtonLink id="back-button" routeId="$lang/_protected/access-to-governmental-benefits/index" params={params} data-gc-analytics-customclick="ESDC-EDSC:CDCP Personal Information:Back - Access to governmental benefits click">
-              {t('access-to-governmental-benefits:access-to-governmental-benefits.view.back')}
-            </ButtonLink>
+  return (
+    <div className="max-w-prose">
+      {updatedInfo && (
+        <ContextualAlert type="success">
+          <h2 className="text-xl font-semibold">{t('access-to-governmental-benefits:access-to-governmental-benefits.view.info-updated.your-info-has-been-updated')}</h2>
+        </ContextualAlert>
+      )}
+      if {hasDentalPlans}
+      {
+        <>
+          <div className="mb-5 space-y-4">
+            {personalInformation.provincialTerritorialDentalPlanId ? (
+              <div className="mb-5 space-y-4">
+                <h2 className="font-bold"> {t('access-to-governmental-benefits:access-to-governmental-benefits.view.provincial-or-territorial-dental-benefit')}</h2>
+                <ul className="list-disc space-y-6 pl-7">
+                  <li>{provincialAndTerritorialProgramName}</li>
+                </ul>
+              </div>
+            ) : (
+              <div className="mb-5 space-y-4">
+                <h2 className="font-bold"> {t('access-to-governmental-benefits:access-to-governmental-benefits.view.provincial-or-territorial-dental-benefit')}</h2>
+              </div>
+            )}
           </div>
-        </fetcher.Form>
-      </div>
-    );
-  } else {
-    return (
-      <div className="max-w-prose">
-        {updatedInfo && (
-          <ContextualAlert type="success">
-            <h2 className="text-xl font-semibold">{t('access-to-governmental-benefits:access-to-governmental-benefits.view.info-updated.your-info-has-been-updated')}</h2>
-          </ContextualAlert>
-        )}
+          <div className="mb-5 space-y-4">
+            {personalInformation.federalDentalPlanId ? (
+              <div className="mb-5 space-y-4">
+                <h2 className="font-bold"> {t('access-to-governmental-benefits:access-to-governmental-benefits.view.federal-benefits-dental-benefits')}</h2>
+
+                <ul className="list-disc space-y-6 pl-7">
+                  <li>{federalSocialProgramName} </li>
+                </ul>
+              </div>
+            ) : (
+              <div className="mb-5 space-y-4">
+                <h2 className="font-bold"> {t('access-to-governmental-benefits:access-to-governmental-benefits.view.federal-benefits-dental-benefits')}</h2>
+              </div>
+            )}
+          </div>
+        </>
+      }
+      :
+      {
```

Comment:
Not sure where to put this comment but when I tested the changes locally, the page renders like this:

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/ae887f6c-6584-40b5-b03a-b6e010a8823b)

See [Figma](https://www.figma.com/design/d8XksFd7eDtBF9V3IvaBeb/CDCP-Client-Portal-(in-MSCA)?node-id=3-4537&t=UVDXOeoEzWAYMJvi-0) for how the page should render.

Reply:
I went back and forth a couple times trying to duplicate this problem, and it turned out that it got fixed when I removed the `if` and switched the parentheses / braces.

So this was fixed in commit 46c8317039d345b42e0ee215c5694e1ad27774ca.